src/main/java/com/moodybluez/enterprise/dao/UserRepository.java

```diff
@@ -1,11 +1,9 @@
 package com.moodybluez.enterprise.dao;
 
 import com.moodybluez.enterprise.dto.User;
-import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
 
 public interface UserRepository extends CrudRepository<User, Integer> {
 
-    @Query("SELECT u FROM User u WHERE u.username = ?1")
```

Comment:
Technical Debt - What you are defining in @ Query annotation is what the CrudRepository method provides built-in, based on what naming convention you use. This could really be confusing for new developers and will spend unnecessary time figuring this out thus adding technical debt. Please see citation to understand CrudRepository naming conventions already available.
Citation - "[Crud Repository Core Concepts](https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#repositories.core-concepts)"